README.md

```diff
@@ -1,37 +1,37 @@
 # Discord.FM
-Discord.FM is a Discord designed for retrieving statistics and data from Last.fm whilest facilitating more community elements through allowing people to link their Last.FM accounts.
+Discord.FM is a Discord bot designed for retrieving statistics and data from Last.fm, whilst facilitating more community elements by allowing people to link their Last.FM accounts.
 
 # Installation
 
 ## Prerequisites
 
-This bot is designed primarily with Python 3.6 in mind in a Linux environment.
+This bot is designed primarily with Python 3.6 and a Linux environment in mind, and may not function properly in other conditions.
 
-To install the neccessary Python modules use the following command in the bot's folder: 
+To install the necessary Python modules, use the following command in the bot's folder (with `sudo` not being required in all environments: 
 
-``pip3 install -r requirements.txt``
+``sudo pip3 install -r requirements.txt``
 
 In-order to enable voice functionality, you need to install ffmpeg3 and keep youtube-dl up-to-date for voice functionality to work using:
```

Comment:
either the start or the end of that line is redundant, "In-order to enable voice functionality"; "for voice functionality to work"